test/JDBC/input/views/sys-check_constraints.sql

```diff
@@ -2,11 +2,12 @@ DROP TABLE IF EXISTS sys_check_constraints
 GO
 
 CREATE TABLE sys_check_constraints (
-	sck_date_col DATETIME CHECK (sck_date_col IS NOT NULL)
+	sck_date_col DATETIME CHECK (sck_date_col IS NOT NULL),
+	sck_datetime_2 datetime2 check(sck_datetime_2 < cast('2020-10-20 09:00:00' as datetime2))
```

Comment:
Could you move this test to `sys-check_constraints-vu` so that it can be tested by both normal mode and `latest->latest MVU`?

Reply:
Updated the tests.